<?xml version="1.0"?>

<launch>
  <node pkg="octomap_server" type="octomap_server_node" name="octomap" output="screen">
    <param name="resolution" value="0.05"/>
    <remap from="/cloud_in" to="/grid_map_visualization/elevation_points"/>
  </node>
</launch>
